README.md

```diff
@@ -1,45 +1,71 @@
-
-## Usage
-
-- Edit the file `suite.yml` as needed
-- Run the CHANGELOG generator:
-```
-./parse-changelogs
-```
-- Resulting changelog will be placed in `CHANGELOG.md`
-
-## Advanced usage
-
-The CLI accepts the following arguments/parameters:
-```
-  -f string
-        Repository YAML file to parse (default "suite.yml")
-  -o string
-        Output filename
-  -p string
-        GitHub API token. This can also be passed in as the 'GITHUB_TOKEN' environment variable. The flag takes precedence.
-  -r string
-        Directory of releases (containinng 'suite_<semver>.yml') files. Set this to empty string to skip suite version diffing. (default "releases")
-  -t string
-        Output type. Only accepts 'changelog', 'docs-release', 'release', and 'unreleased'. (default "changelog")
-  -v string
-        Version to embed in the changelog (default "Unreleased")
-```
+![Tests](https://github.com/cyberark/conjur-oss-suite-release/workflows/Tests/badge.svg)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/31060f348b29c7f5d02b/test_coverage)](https://codeclimate.com/repos/5e2b43bf92af05714c00b172/test_coverage)
+[![Maintainability](https://api.codeclimate.com/v1/badges/31060f348b29c7f5d02b/maintainability)](https://codeclimate.com/repos/5e2b43bf92af05714c00b172/maintainability)
+
+## What is the Conjur OSS Suite?
+
+[CyberArk Conjur](https://github.com/cyberark/conjur) is a RESTful web service that
+can be used to securely authenticate, control and audit non-human access across
+tools, applications, containers and cloud environments via robust secrets management.
+
+To ensure that Conjur is easy to use no matter your application's language, the
+tools you use to build or deploy it, or the platform you deploy it to, we've built
+an extensive set of additional plugins, libraries, and software that extend Conjur
+to work natively with a variety of tools and in a wide range of environments.
+
+The **Conjur OSS Suite** aggregates this set of tools to provide a _single place_
+where you can:
+- Find out about all of the open source tools and integrations that work with Conjur
+- Learn about new features in existing components
+- Learn about new integrations or tools that work with Conjur
+- Find out which version of Conjur to use, and which corresponding versions of
+  the suite components are compatible with it
+
+If you're using Conjur OSS, we **strongly recommend** that you reference the latest
+suite release (link TBA) to determine which version of Conjur to use, and which corresponding
+versions of the suite are compatible with it.
+
+### A note on versioning
+The core of each Conjur OSS Suite release is the [Conjur server](https://github.com/cyberark/conjur).
```

Comment:
Do we use the term `server` elsewhere for Conjur?  Feels like slightly odd word.

Reply:
yes, again on cyberark.github.io/conjur. `core` and `server` are both terms PM has used to describe the oss architecture; `core` is the label and `server` is just descriptive, as it is a literal server